packages/common-all/src/error.ts

```diff
@@ -16,10 +16,9 @@ type DendronErrorProps = {
   severity?: ERROR_SEVERITY;
 
   /**
-   * @deprecated - should only used in DendronServerError
    * Optional HTTP status code for error
```

Comment:
Can you amend this description to "Optional TCode to identify the type of error" or something to that effect? Basically, remove the 'HTTP' reference.  Or, maybe mention that it'll be a Status_Code if you don't specify a generic parameter.

Was `<TCode = StatusCodes | undefined>` added for compatibility purposes?

Reply:
> Was `<TCode = StatusCodes | undefined>` added for compatibility purposes?

Yes, that's what it used to be, so I wanted to keep that backwards compatible.